community-server/src/main/resources/application.properties

```diff
@@ -127,3 +127,20 @@ management.metrics.export.datadog.enabled=false
 #optional - default Datadog instance is https://app.datadoghq.com/ 
 # management.metrics.export.datadog.uri=<your DD instance>
 #management.metrics.export.datadog.step=10s
+
+#Following properties set for using for publishing workflow/task status change event to RabbitMQ:
```

Comment:
Style: Looking around, I see that "non-code" comments should have a space between `#` and the sentence (not unlike our own [.NET Coding Conventions](https://github.com/codaxy/curriculum/blob/master/dotnet/conventions.md))